packages/connect-examples/expo-example/public/index.html

```
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
```

HTMLHint (1.5.0)

[error] The attribute name of [ httpEquiv ] must be in lowercase.

(attr-lowercase)